src/clients/SpokePoolClient.ts

```diff
@@ -663,7 +827,21 @@ export class SpokePoolClient {
     return this.hubPoolClient.getDestinationTokenForDeposit(deposit);
   }
 
+  /**
+   * Performs a log for a specific level, message and data.
+   * @param level The log level.
+   * @param message The log message.
+   * @param data Optional data to log.
+   */
   protected log(level: DefaultLogLevels, message: string, data?: AnyObject) {
     this.logger[level]({ at: "SpokePoolClient", chainId: this.chainId, message, ...data });
   }
+
+  /**
+   * Retrieves the current time from the SpokePool contract.
+   * @returns The current time.
+   */
+  public getCurrentTime(): number {
+    return this.currentTime;
+  }
```

Comment:
@pxrl I've included this for convenience. This satisfies the comment left in: https://github.com/across-protocol/relayer-v2/pull/787/files